protocol/daemons/server/server.go

```diff
@@ -22,32 +27,73 @@ type Server struct {
 	fileHandler   lib.FileHandler
 	socketAddress string
 
+	updateMonitor *types.UpdateMonitor
+
 	BridgeServer
 	PriceFeedServer
 	LiquidationServer
 }
 
 // NewServer creates a single gRPC server that's shared across multiple daemons for communication.
+// cosmosGrpcServiceAddress is the address of the Cosmos gRPC query service. This address is used to uniquely
```

Comment:
mismatch between comment `cosmosGrpcServiceAddress` and parameter name `uniqueTestIdentifier`

Reply:
Sorry about that, I missed that in my refactor!